GitUI/BranchTreePanel/RepoObjectsTree.Designer.cs

```diff
@@ -626,24 +557,27 @@ private void InitializeComponent()
         private Button btnSettings;
         private ContextMenuStrip menuSettings;
         private ToolStripMenuItem showTagsToolStripMenuItem;
-        private ToolStripMenuItem mnubtnMergeBranch;
-        private ToolStripMenuItem mnubtnRebase;
         private ToolStripSeparator toolStripSeparator1;
-        private ToolStripSeparator toolStripSeparator2;
-        private ToolStripMenuItem mnubtnReset;
         private ToolStripMenuItem mnubtnManageRemotes;
         private ToolStripMenuItem mnuBtnManageRemotesFromRootNode;
         private ToolStripMenuItem mnubtnRemoteBranchFetchAndCheckout;
         private ToolStripMenuItem mnuBtnCheckoutTag;
         private ToolStripMenuItem mnubtnFetchRebase;
         private ToolStripMenuItem mnubtnFetchCreateBranch;
         private ToolStripSeparator tsmiSpacer1;
-        private ToolStripSeparator tsmiSpacer2;
         private ToolStripSeparator tsmiSpacer4;
         private ToolStripMenuItem mnubtnEnableRemote;
         private ToolStripMenuItem mnubtnEnableRemoteAndFetch;
         private ToolStripMenuItem mnubtnDisableRemote;
         private ToolStripSeparator tsmiSpacer3;
         private ToolTip toolTip;
+        /// <summary>
+        /// Local branch context menu [check out] actions group
+        /// </summary>
+        private CheckoutMenuItemsGroup<LocalBranchNode> _localBranchCheckoutMenuItems;
+        /// <summary>
+        /// Remote branch context menu [check out] actions group
+        /// </summary>
+        private CheckoutMenuItemsGroup<RemoteBranchNode> _remoteBranchCheckoutMenuItems;
```

Comment:
This file is auto-generated and for the most part auto-updated, manual changes should be avoided in this file and placed in the main (i.e. non designer) file.